ls1intum/Artemis
# [Feature] Feature Overview for Exam Mode

### Checklist

### Motivation and Context
We want to create a page which displays the features of the exam mode in a clear way.
We also made some changes to the exam end page to reduce the clutter. 

### Description
- Create the general structure of the view. 
- Create different routes for students and instructors. 
- Lazy load the features overview module
- Create a pre-configured structure for texts including 1 text, 1 image, 2 texts, 2 images, centre text, first image left, first image right
- Add the features displayed in: 

> https://confluence.ase.in.tum.de/display/ArTEMiS/Feature+Overview+German
> https://confluence.ase.in.tum.de/display/ArTEMiS/Feature+Overview+English

- Add the text for each feature
- Add the screenshots where relevant

### Coverage
feature-overview.component.ts 67.5%


### Steps for Testing
1. nagivate to /#/features/instructors
2. navigate to /#/features/students
When you hover over a feature, more information is displayed.
When you press on a feature you should automatically scroll to the feature details.

### Screenshots
![image](https://user-images.githubusercontent.com/15110960/95776010-c64ca580-0cc3-11eb-868b-b50f8116089e.png)
![image](https://user-images.githubusercontent.com/15110960/95776027-ce0c4a00-0cc3-11eb-996c-ecd7ba6292e6.png)
![image](https://user-images.githubusercontent.com/15110960/95776041-d49ac180-0cc3-11eb-9898-f0b53bc42bdd.png)
